hilda/ipython_extensions/keybindings.py

```diff
@@ -0,0 +1,28 @@
+from prompt_toolkit.enums import DEFAULT_BUFFER
+from prompt_toolkit.filters import EmacsInsertMode, HasFocus, HasSelection, ViInsertMode
+from prompt_toolkit.keys import Keys
+
+
+def load_ipython_extension(ipython):
+    def register_keybindings():
+        hilda = ipython.user_ns.get('p')
+        if not hilda:
+            print("Hilda instance not found in IPython namespace.")
```

Comment:
raise instead

Reply:
Changed it to access `ipython.user_ns['p']` exception will raise if no exsists